media/windows/ECC-Ecobee/ECC Pycobee Data.py

```diff
@@ -149,6 +149,23 @@
     Also corrected bug in processing of GMail credentials file and included some missing
     imports for the new mail routine for error-aborts.
             Modified by DK Fowler ... 06-Dec-2020       --- v03.11
+<<<<<<< Updated upstream
+=======
```

Comment:
I think these conflict markers accidentally got left in.  There's a few of these in the commit; you might want to resolve all of them and push an update.

Reply:
Argh!  I'm not sure how to go about that but will see if I can figure it out...

Reply:
It's easiest to do it as a 2-step process:

1. Resolve / fix the conflict markers, and then commit the fix.  This makes a 2nd commit on the branch (and if you push to github, a 2nd commit on the PR).
2. Then do an interactive rebase: `git rebase -i origin/master`.  That will bring up an editor with all the commits on your branch (i.e., the 2 of them).  From there, you can squash the two commits together, and you'll be left with a single commit on your branch (FWIW: I do this All. The. Time. on my branches -- interactive rebases [effectively] allow me to edit / slice / dice my commits until I get them exactly as I want them).  Then you can push this back up to github, and the PR will update.
    * **NOTE:** You'll need to `git push --force ...` to push back up to github, because a rebase will change git history, and github will disallow that by default.  It's basically github's way of saying "Hey, you changed git history... are you sure you want to push and overwrite the history you have here at github?"  And `--force` is your way of saying "Yes, Github, I'm sure."

Reply:
A more basic question…
I assume I first need to revert the last commit with “git reset —soft HEAD^”?  Then the “git commit…” to get edit / get rid of the conflict markers, then the “git rebase…”, followed by the “git push…”?  Just want to make sure I have it straight…

Reply:
I'd also like to understand precisely how your (first) recommendation would work.  So, when you say...

> Resolve / fix the conflict markers, and then commit the fix. This makes a 2nd commit on the branch (and if you push to github, a 2nd commit on the PR).

How would I do this first edit / fix without first doing a "git reset..."?  Wouldn't the (local) working copy be "clean" without the conflict markers?  So, nothing to fix?  Sorry, I'm missing something here I think...

Reply:
Just to close out the github thread: we talked this through on a Webex + Slack.  Resolved.